src/Extension/Traits/FluentObjectTrait.php

```diff
@@ -109,12 +115,48 @@ protected function updateFluentCMSFields(FieldList $fields)
             'Locale' => 'Locale'
         ];
 
+        // Augment Localisation tab with clickable locale links to allow easy navigation between model localisations
+        if ($owner->hasExtension(CMSEditLinkExtension::class)) {
+            $columns->setFieldCasting([
+                // Make sure our links are correctly rendered (valid HTML)
+                'Title' => 'HTMLVarchar->RAW',
+            ]);
+
+            $controller = Controller::has_curr() ? Controller::curr() : null;
+            $request = $controller?->getRequest();
+
+            // Pass getVars separately so we can process them later
+            $params = $request?->getVars() ?? [];
+
+            // This is to get URL only, getVars are not part of the URL
+            $url = $owner->CMSEditLink();
+            $url = Director::makeRelative($url);
+
+            $summaryColumns['Title'] = [
+                'title' => 'Title',
+                'callback' => function (Locale $object) use ($url, $params) {
+                    if (!$object->RecordLocale()) {
+                        return null;
+                    }
+
+                    $recordLocale = $object->RecordLocale();
+                    $locale = $recordLocale->getLocale();
+                    $params['l'] = $locale;
+                    $localeLink = Controller::join_links($url, '?' . http_build_query($params));
+                    $localeTitle = Convert::raw2xml($recordLocale->getTitle());
```

Comment:
Presumably this is there to sure that someone couldn't put a XSS payload in the Locale Title.

Are we happy with the assumption that there's no XSS vector in the locale link?

Reply:
I would say that this is out of scope of this PR. Not really changing how the link is generated here.

Reply:
Yup it's largely a copy/paste. The only real change for the purposes of what's rendered is that it's not being handled by `DBHTMLVarchar` anymore, and that class didn't do any sanitisation. This PR doesn't introduce any risks that weren't already there.